javaCreateAccount/src/main/java/com/hamdan/forzenbook/java/createaccount/JavaCreateAccountActivity.java

```diff
@@ -40,11 +40,13 @@ protected void onCreate(Bundle savedInstanceState) {
         FrameLayout view = binding.getRoot();
         setContentView(view);
 
+        model.onSuccess = () -> model.succeedCreation(JavaCreateAccountActivity.this);
```

Comment:
There shouldn't be lambdas, there should  be RX singles

Reply:
its a lambda that is called inside the rx single, it's called on succesfully creating the account inside the single with the purpose being for navigation.

Reply:
removed